lib/ecto/migration/runner.ex

```diff
@@ -16,7 +17,12 @@ defmodule Ecto.Migration.Runner do
   """
   def run(repo, module, direction, operation, migrator_direction, opts) do
     level = Keyword.get(opts, :log, :info)
-    start_link(repo, direction, migrator_direction, level)
+    ensure_manager_started()
```

Comment:
I've got this set up in the latest commit, and it starts the manager fine when using iex -S mix. However, in the tests, I get a flood of errors about no process, at the call to put_migration in the test setup. The app should be up at this point, what might be the cause of this?